src/renderer/src/components/OverviewCharts.tsx

```diff
@@ -27,30 +28,12 @@ export const OverviewCharts = ({
   balanceChartClassName,
   chartCardListClassName,
 }: TProps) => {
-  const sortedBalances = useMemo(() => {
-    const tokensBalances = balances.data.map(balance => balance.tokensBalances).flat()
-
-    const map = new Map<string, TTokenBalance>()
-
-    tokensBalances.forEach(balance => {
-      if (balance.exchangeAmount <= 0) return
-
-      const repeated = map.get(balance.token.hash)
-      if (repeated) {
-        repeated.amountNumber += balance.amountNumber
-        repeated.exchangeAmount += balance.exchangeAmount
-        repeated.amount = BSBigNumberHelper.format(repeated.amountNumber)
-        return
-      }
-
-      map.set(balance.token.hash, cloneDeep(balance))
-    })
-
-    return Array.from(map.values()).sort((token1, token2) => token2.exchangeAmount - token1.exchangeAmount)
-  }, [balances])
+  const sortedBalances = balances.groupedTokenBalances.sort(
```

Comment:
Use:
```suggestion
  const sortedBalances = balances.groupedTokenBalances.toSorted(
```

Or try to use a Lodash method.